starterFiles/components/avatar.css

```diff
@@ -0,0 +1,34 @@
+@import url('../abstracts/variables.css');
+@import url('../base/reset.css');
+@import url('../utils/util.css');
+@import url('../utils/margin.css');
+@import url('../utils/padding.css');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+/* avatar */
+.avatar-small {
+  width: 2rem;
+  height: 2rem;
+}
+
+.content-wrapper {
```

Comment:
could rename this class to `avatar-wrapper` to keep the consistency or put this class to some global file like helpers.